<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Nouveau"/>
    <addaction name="action_Ouvrir"/>
    <addaction name="action_Fermer"/>
    <addaction name="separator"/>
    <addaction name="action_Enregistrer"/>
    <addaction name="action_Enregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="action_Imprimer"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edition">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="action_Annuler"/>
    <addaction name="separator"/>
    <addaction name="action_Couper"/>
    <addaction name="action_Copier"/>
    <addaction name="action_Coller"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Affichage">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
    <addaction name="actionZoom_avant"/>
    <addaction name="action_Zoom_arriere"/>
   </widget>
   <widget class="QMenu" name="menu_Image">
    <property name="title">
     <string>&amp;Image</string>
    </property>
    <addaction name="actionRecadrer"/>
    <addaction name="separator"/>
    <addaction name="actionRotation_180"/>
    <addaction name="actionRotation_90_Horaire"/>
    <addaction name="action_Rotation_90_antihoraire"/>
   </widget>
   <widget class="QMenu" name="menu_Effet">
    <property name="title">
     <string>&amp;Effet</string>
    </property>
    <widget class="QMenu" name="menu_Flou">
     <property name="title">
      <string>&amp;Flou</string>
     </property>
     <addaction name="action_Flou_gaussien"/>
     <addaction name="action_Flou_moyenneur"/>
     <addaction name="action_Personnalis_flou"/>
     <addaction name="separator"/>
     <addaction name="action_Taille_filtre"/>
    </widget>
    <widget class="QMenu" name="menu_Datetction_de_contours">
     <property name="title">
      <string>&amp;Datetction de contours</string>
     </property>
     <addaction name="action_Flou_Sobel"/>
     <addaction name="action_Flou_Roberts"/>
     <addaction name="action_Personnalis_Detec"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_Flou"/>
    <addaction name="menu_Datetction_de_contours"/>
   </widget>
   <widget class="QMenu" name="menu_Fen_tre">
    <property name="title">
     <string>&amp;Fenêtre</string>
    </property>
    <addaction name="actionFusion"/>
    <addaction name="actionBarre_d_outil"/>
    <addaction name="actionBarre_de_menu"/>
    <addaction name="actionCouleurs"/>
    <addaction name="actionHistogramme"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menu_Edition"/>
   <addaction name="menu_Affichage"/>
   <addaction name="menu_Image"/>
   <addaction name="menu_Effet"/>
   <addaction name="menu_Fen_tre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Nouveau"/>
   <addaction name="action_Ouvrir"/>
   <addaction name="action_Enregistrer"/>
   <addaction name="separator"/>
   <addaction name="action_Imprimer"/>
   <addaction name="separator"/>
   <addaction name="action_Couper"/>
   <addaction name="action_Copier"/>
   <addaction name="action_Coller"/>
   <addaction name="action_Annuler"/>
   <addaction name="actionValider"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Selection"/>
   <addaction name="actionDeplacement"/>
   <addaction name="actionPipette"/>
   <addaction name="actionCrop"/>
   <addaction name="actionFlou"/>
   <addaction name="actionSeamCarving"/>
   <addaction name="actionInverseCoul"/>
   <addaction name="actionImageGris"/>
   <addaction name="actionFusion_2"/>
   <addaction name="actionHistogramme_2"/>
   <addaction name="actionZoom_avant"/>
  </widget>
  <action name="action_Nouveau">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/new file.png</normaloff>:/Edition/Menu Edition/new file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Ouvrir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/Open.png</normaloff>:/Edition/Menu Edition/Open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Fermer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/Close file.png</normaloff>:/Edition/Menu Edition/Close file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fermer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Enregistrer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/Save.png</normaloff>:/Edition/Menu Edition/Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Enregistrer_sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/Save as.png</normaloff>:/Edition/Menu Edition/Save as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Enregistrer sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Imprimer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Edition/Menu Edition/Print.png</normaloff>:/Edition/Menu Edition/Print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Imprimer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Annuler">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/File/Menu File/Undo.png</normaloff>:/File/Menu File/Undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Couper">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/File/Menu File/couper.png</normaloff>:/File/Menu File/couper.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Couper</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copier">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/File/Menu File/copier.png</normaloff>:/File/Menu File/copier.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Coller">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/File/Menu File/Paste.png</normaloff>:/File/Menu File/Paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Coller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionZoom_avant">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Affichage/Menu Afichage/Zoom avant.png</normaloff>:/Affichage/Menu Afichage/Zoom avant.png</iconset>
   </property>
   <property name="text">
    <string>Zoom avant</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_Zoom_arriere">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Affichage/Menu Afichage/Zoom arriere.png</normaloff>:/Affichage/Menu Afichage/Zoom arriere.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom arrière</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionRotation_90_Horaire">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/Menu Image/Rotation clock.png</normaloff>:/Image/Menu Image/Rotation clock.png</iconset>
   </property>
   <property name="text">
    <string>Rotation 90° Horaire</string>
   </property>
  </action>
  <action name="action_Rotation_90_antihoraire">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/Menu Image/Rotation otherclock.png</normaloff>:/Image/Menu Image/Rotation otherclock.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotation 90° antihoraire</string>
   </property>
  </action>
  <action name="actionRotation_180">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/Menu Image/Rotation 180.png</normaloff>:/Image/Menu Image/Rotation 180.png</iconset>
   </property>
   <property name="text">
    <string>Rotation 180°</string>
   </property>
  </action>
  <action name="action_Flou_gaussien">
   <property name="text">
    <string>&amp;Flou gaussien </string>
   </property>
  </action>
  <action name="action_Flou_moyenneur">
   <property name="text">
    <string>&amp;Flou moyenneur</string>
   </property>
  </action>
  <action name="action_Personnalis_flou">
   <property name="text">
    <string>&amp;Personnalisé...</string>
   </property>
  </action>
  <action name="action_Flou_Sobel">
   <property name="text">
    <string>&amp;Flou Sobel</string>
   </property>
  </action>
  <action name="action_Flou_Roberts">
   <property name="text">
    <string>&amp;Flou Roberts</string>
   </property>
  </action>
  <action name="action_Personnalis_Detec">
   <property name="text">
    <string>&amp;Personnalisé...</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionBarre_d_outil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre d'outil</string>
   </property>
  </action>
  <action name="actionBarre_de_menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre de menu</string>
   </property>
  </action>
  <action name="actionCouleurs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Couleurs</string>
   </property>
  </action>
  <action name="actionHistogramme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Histogramme</string>
   </property>
  </action>
  <action name="action_Taille_filtre">
   <property name="text">
    <string>&amp;Taille filtre</string>
   </property>
  </action>
  <action name="action_Selection">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Selection.png</normaloff>:/Barre outil/Tool bar/Selection.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Selection</string>
   </property>
  </action>
  <action name="actionDeplacement">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Deplacement.png</normaloff>:/Barre outil/Tool bar/Deplacement.png</iconset>
   </property>
   <property name="text">
    <string>Deplacement</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Crop.png</normaloff>:/Barre outil/Tool bar/Crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionFlou">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Flou.png</normaloff>:/Barre outil/Tool bar/Flou.png</iconset>
   </property>
   <property name="text">
    <string>Flou</string>
   </property>
  </action>
  <action name="actionSeamCarving">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Seamcarving.png</normaloff>:/Barre outil/Tool bar/Seamcarving.png</iconset>
   </property>
   <property name="text">
    <string>SeamCarving</string>
   </property>
  </action>
  <action name="actionPipette">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/group1/pipette-icon.png</normaloff>:/group1/pipette-icon.png</iconset>
   </property>
   <property name="text">
    <string>Pipette</string>
   </property>
  </action>
  <action name="actionInverseCoul">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/negatif couleur.png</normaloff>:/Barre outil/Tool bar/negatif couleur.png</iconset>
   </property>
   <property name="text">
    <string>InverseCoul</string>
   </property>
  </action>
  <action name="actionImageGris">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/imagegris.png</normaloff>:/Barre outil/Tool bar/imagegris.png</iconset>
   </property>
   <property name="text">
    <string>ImageGris</string>
   </property>
  </action>
  <action name="actionFusion_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Fusion.png</normaloff>:/Barre outil/Tool bar/Fusion.png</iconset>
   </property>
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionRecadrer">
   <property name="text">
    <string>Recadrer</string>
   </property>
  </action>
  <action name="actionValider">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/valider.png</normaloff>:/Barre outil/Tool bar/valider.png</iconset>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </action>
  <action name="actionHistogramme_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Barre outil/Tool bar/Histogramme.png</normaloff>:/Barre outil/Tool bar/Histogramme.png</iconset>
   </property>
   <property name="text">
    <string>Histogramme</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
